docs/pages/connect-your-client/model-context-protocol/mcp-access.mdx

```diff
@@ -10,9 +10,17 @@ Teleport.
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- The Teleport MCP Access configured. See our guides for how to set up the [MCP
-  Access](../../enroll-resources/mcp-access/mcp-access.mdx).
+- The Teleport MCP Access configured.
 
+  <details open>
+  <summary>Configure MCP Access</summary>
+
+  - [Quick-start with demo MCP server](../../enroll-resources/mcp-access/getting-started.mdx).
+
+  - [MCP Access with Stdio MCP Server](../../enroll-resources/mcp-access/stdio.mdx)
```

Comment:
i really like quick-start link here. Stdio is fine for now but it might need to change once we add support for other transports. (to avoid maintaining that) I suggest we just link the higher level page like `[Enrol MCP servers](...`. WDYT

Reply:
I agree that having to update once we support alternative configuration methods might not be ideal. But by having this here, we save the users one click that might get them lost (as the getting started pages have ~6 options for them to go for).

What do you think of keeping this here and adding the getting started link to the bullet point, like:

```
Teleport [MCP Access](getting-started.mdx) configured.
```